### ISSUE TITLE
Fix for filename convention 

### ISSUE THREAD

**Issue by dbekaert** [dbekaert/RAiDER#129]
hdf5 file always refers from East "E". Test with -b option such as '39 40 -79 -78' and '39 40 78 79'

**Comment by dbekaert** [comment]
Issue not yet resolved in latest dev version:

```
(raider) MT-101939:RAiDER-most-current dbekaert$ ls weather_files/
ERA-5T_2020_01_02_T12_00_00.nc				ERA5T_2020-01-02T12_00_00_-10.0N_-9.0N_-1.0E_1.0E.h5
ERA-5T_2020_01_04_T12_00_00.nc				ERA5T_2020-01-04T12_00_00_-10.0N_-9.0N_-1.0E_1.0E.h5
ERA-5T_2020_01_06_T12_00_00.nc				ERA5T_2020-01-06T12_00_00_-10.0N_-9.0N_-1.0E_1.0E.h5
ERA-5T_2020_01_08_T12_00_00.nc				ERA5T_2020-01-08T12_00_00_-10.0N_-9.0N_-1.0E_1.0E.h5
ERA-5T_2020_01_10_T12_00_00.nc				ERA5T_2020-01-10T12_00_00_-10.0N_-9.0N_-1.0E_1.0E.h5
ERA-5_2020_01_03_T23_00_00.nc				ERA5_2020-01-03 23:00:00_39.0N_40.0N_-79.0E_-78.0E.h5
```

**Comment by dbekaert** [comment]
@leiyangleon could you have a look in this?

Few thoughts:

1. The download of the models if a users users 0 to 360 or -180 to 180; we should have a consistent filename of either 180W to 180E; would recommend to avoid -180 in a filename.

2. We should ensure that what we feed into API for ECMWF, PyDAP for GMAO, HRES etc is according to what the API support. e.g. can they handle both 0 to 360 or does it need to be -180 to 180.

**Comment by jlmaurer** [comment]
Closed with #161.
